standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestDatabases.java

```diff
@@ -454,6 +454,22 @@ public void testGetDatabases() throws Exception {
     databases = client.getDatabases("*_not_such_database_*");
     Assert.assertEquals("No such databases size", 0, databases.size());
 
+    // Find all the databases by using * as the database names' pattern
+    databases = client.getDatabases("*");
```

Comment:
We don't make any changes on the `getDatabases` method, these tests look unrelated to the change.